A/ApproxFunFourier/Compat.toml

```diff
@@ -11,13 +10,22 @@ DualNumbers = "0.4-0.6"
 FFTW = "0.2.4"
 FastGaussQuadrature = "0.3.2-0.3.3"
 FastTransforms = "0.4.2"
-FillArrays = "0.5"
 InfiniteArrays = "0.0.3"
 IntervalSets = "0.3.1"
-LazyArrays = "0.7.1-0.8"
 LowRankApprox = "0.1.4-0.2"
 Reexport = "0.2"
 SpecialFunctions = "0.7"
 StaticArrays = "0.8.3-0.10"
 ToeplitzMatrices = "0.4-0.5"
+
+["0.0.1"]
+ApproxFunBase = "0.0.2-0.0.3"
+FillArrays = "0.5"
+LazyArrays = "0.7.1-0.8"
 julia = "0.7-1.3"
+
+["0.0.2"]
+ApproxFunBase = "0.0.3-0.0.4"
+FillArrays = "0.5-0.6"
+LazyArrays = "0.8"
+julia = "0.7-1.1"
```

Comment:
This is wrong, see https://github.com/JuliaComputing/Registrator.jl/issues/103. Linking it with that issue so we can correct it later.